src/jvm/test/resources/expected_logs/suspend_trace_reporting.txt

```diff
@@ -44,124 +44,124 @@ The following interleaving leads to the error:
+|                                                                                                                                                                                 |                   _state.get(): SemaphoreSegment#1 at CancellableContinuationImpl.tryResumeImpl(CancellableContinuationImpl.kt:0)                                                      |
+|                                                                                                                                                                                 |                   resumeMode ➜ 1 at CancellableContinuationImpl.tryResumeImpl(CancellableContinuationImpl.kt:540)                                                                      |
+|                                                                                                                                                                                 |                   _state.compareAndSet(SemaphoreSegment#1, CompletedContinuation#1): true at CancellableContinuationImpl.tryResumeImpl(CancellableContinuationImpl.kt:541)             |
+|                                                                                                                                                                                 |                   detachChildIfNonResuable() at CancellableContinuationImpl.tryResumeImpl(CancellableContinuationImpl.kt:542)                                                          |
+|                                                                                                                                                                                 |                     isReusable(): false at CancellableContinuationImpl.detachChildIfNonResuable(CancellableContinuationImpl.kt:565)                                                    |
+|                                                                                                                                                                                 |                       resumeMode ➜ 1 at CancellableContinuationImpl.isReusable(CancellableContinuationImpl.kt:141)                                                                     |
+|                                                                                                                                                                                 |                     detachChild$kotlinx_coroutines_core() at CancellableContinuationImpl.detachChildIfNonResuable(CancellableContinuationImpl.kt:565)                                  |
+|                                                                                                                                                                                 |                       getParentHandle(): ChildContinuation#1 at CancellableContinuationImpl.detachChild$kotlinx_coroutines_core(CancellableContinuationImpl.kt:572)                    |
```

Comment:
Should we show `$kotlinx_coroutines_core` here?

Reply:
Isn't this something that should be resolved with silent sections?

Reply:
We still need to support analyzing coroutine internals

Reply:
Done